typescript/sdk/src/proxy.ts

```diff
@@ -27,10 +27,7 @@ export function isProxyAddresses(
   );
 }
 
-export interface BeaconProxyAddresses
-  extends ProxyAddresses<ProxyKind.UpgradeBeacon> {
-  beacon: types.Address;
-}
+export type TransparentProxyAddresses = ProxyAddresses<ProxyKind.Transparent>;
```

Comment:
would have expected the admin address here if its required everywhere
though I see how it could be optional

Reply:
It felt redundant given we had a separate entry for the proxy admin. I don't feel strongly